MatthewKelly12/here_we_grow_django_py
# changed int to string in return for Condition model

## Link to Ticket
Closes
 
## Description of Proposed Changes
-changed int to string in return of Condition model

## Steps to Test

Outline the steps to test

```
git fetch --all
git checkout conditon

```

1. Install python 3.6 virtual environment and activate it
1. make sure Django 2.1.1 is installed (version indicated in myvenv)
1. python manage.py migrate && python manage.py createsuperuser
1. python manage.py runserver
1. visit page http://localhost:8000/admin/temp_test/condition/add/
1. attempt to add Condition object via admin
1. verify Condition added at  http://localhost:8000/admin/temp_test/condition/

## Impacted Areas in Application

- Condition Model
